lib/typescript/render/providers/facebook/facebookModel.ts

```diff
@@ -14,7 +50,7 @@ export interface ButtonAttachment extends Attachment {
   payload: {
     text: string;
     template_type: 'button';
-    buttons: Button[];
+    buttons: (URLButton | PostbackButton | CallButton | LoginButton | LogoutButton | GamePlayButton)[];
   };
```

Comment:
Fixed the typing here: the messages templates include a `buttons` array that can accept different buttons (and not only web_url buttons)

https://developers.facebook.com/docs/messenger-platform/reference/buttons